README.md

```diff
@@ -12,3 +12,50 @@ This is a helper package that automatically installs and updates the
 
 Open configuration file using command palette with `Preferences: LSP-volar Settings` command or opening it from the Sublime menu (`Preferences > Package Settings > LSP > Servers > LSP-volar`).
 
+### Take over mode
```

Comment:
Can we not call it "take over". It really has little meaning in context of ST.

Reply:
While I would not mind a different name, 

I think a lot of Volar user already got used to the name "take over mode", so that is the only reason why I would keep the name.
https://github.com/johnsoncodehk/volar#ide-supports-by-community


Here is my suggestion:
```diff
- ### Take over mode
+ ### Enable Volar outside of vue files
```

Not very creative, but is self explanatory.
Do you have something better?

Reply:
`Enable for non-Vue files` maybe?

Skipped `Volar` since the context is clear.

I think that if we just skip the "take over mode" term completely, it will will be much more clear to the ST users.